GameData/RealSolarSystem/Compatibility/CustomAsteroids.cfg

```diff
@@ -673,4 +673,297 @@ AsteroidSets
 			max = -2
 		}
 	}
+	
+	ASTEROIDGROUP
+	{
+		name = innerOort
+		title = Inner Oort Cloud Obj.
+
+		centralBody = Sun
+
+		// One object every 710 Earth days or about 2 years
+		spawnRate = 0.00141
+
+		orbitSize
+		{
+			type = SemimajorAxis
+			min = Ratio(Earth.sma, 2500)
+			max = Ratio(Earth.sma, 20000)
+		}
+
+		eccentricity
+		{
+			dist = Uniform
+			min = 0.1
+			max = 0.999
+		}
+
+		inclination
+		{
+			dist = Uniform
+			min = -20
+			max = 20
+		}
+
+		asteroidTypes
+		{
+			key = 0.8 CaAsteroidIcy
+			// key = 0.1 CaCometActive
+			key = 0.18 PotatoComet // CaCometInterstellar
```

Comment:
@Starstrider42 I just put `CaCometInterstellar` because I had an issue with a previous version of RSS and custom asteroids with the science samples, that was just an oversight that I forgot to remove.